src/java.base/share/classes/java/util/Locale.java

```diff
@@ -232,6 +233,51 @@
  * implementations in a Java Runtime Environment might not support any
  * particular Unicode locale attributes or key/type pairs.
  *
+ * <h2><a id="t_extension">Transformed Content extension</a></h2>
+ * <a href="https://datatracker.ietf.org/doc/html/rfc6497">RFC 6497</a>
+ * specifies an extension to BCP 47 that provides subtags for specifying
+ * the source language or script of transformed content. Transformed content
+ * is content that has been transformed, including text that has been
+ * transliterated, transcribed, translated, or in some other way
+ * influenced by the source locale. For example,
+ * <table class="striped">
+ * <caption style="display:none">Transformed Content extension examples</caption>
+ * <thead>
+ * <tr><th scope="col">Language Tag</th>
+ *     <th scope="col">Description</th></tr>
+ * </thead>
+ * <tbody>
+ * <tr><th scope="row" style="text-align:left">ja-t-it</th>
+ *     <td>The content is Japanese, transformed from Italian.</td></tr>
+ * <tr><th scope="row" style="text-align:left">ja-Kana-t-it</th>
+ *     <td>The content is Japanese Katakana transformed from Italian.</td></tr>
+ * <tr><th scope="row" style="text-align:left">und-Latn-t-und-cyrl</th>
+ *     <td>The content is in the Latin script, transformed from the Cyrillic script.</td></tr>
+ * <tr><th scope="row" style="text-align:left">und-Cyrl-t-und-latn-m0-ungegn-2007</th>
+ *     <td>The content is in Cyrillic, transformed from Latin, according to a UNGEGN specification dated 2007.</td></tr>
+ * </tbody>
+ * </table>
+ * <p>The transformed content extension contains an optional well-formed BCP47 {@code source}
+ * language tag followed by zero or more {@code field}s. At least the {@code source} language tag
+ * or one {@code field} must be included. See the example above. Each {@code field} consists of a
+ * field separator (one alpha + one digit), followed by one or more subtags of the length 3 to 8,
+ * each delimited by a hyphen.
+ * <p>The transformed content information {@code source} language tag and {@code fields} are returned
```

Comment:
"transformed content information" seems not needed as "The {@code source} language tag and {@code fields}" are known from the previous paragraph.

Reply:
Modified to make it more sense.